### ISSUE TITLE
BUG: Pasting a button over an existing button inherits buttons actions

### ISSUE THREAD

**Issue by v4u6h4n** [StreamController/StreamController#76]
If I copy and paste button A onto button B, the new button keeps the actions of button B.

**Comment by Core447** [comment]
Should be fixed with b212016c63129506dfda5d407f9ea8ec27bf47cb. Can you please try again?

**Comment by v4u6h4n** [comment]
Not sure if it was this, or another commit, but it won't startup anymore.

EDIT: Reverted to `b9b2c99` (picked randomly) and it started up fine.

```
2024-04-28 09:12:09.333 | INFO     | __main__:quit_running:172 - Checking if another instance is running
2024-04-28 09:12:09.333 | INFO     | __main__:quit_running:180 - No other instance running, continuing
2024-04-28 09:12:09.333 | ERROR    | __main__:quit_running:181 - org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
2024-04-28 09:12:10.411 | ERROR    | src.backend.GnomeExtensions:connect_dbus:31 - Failed to connect to D-Bus: org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
2024-04-28 09:12:10.429 | INFO     | src.backend.PageManagement.PageManagerBackend:update_auto_change_info:325 - Updated auto-change info in 0.009502649307250977 seconds

(process:128429): Gtk-WARNING **: 09:12:10.694: Unknown key gtk-modules in /home/v4u6h4n/.config/gtk-4.0/settings.ini
2024-04-28 09:12:10.805 | ERROR    | __main__:<module>:251 - An error has been caught in function '<module>', process 'MainProcess' (128429), thread 'MainThread' (134255388088128):
Traceback (most recent call last):

> File "/home/v4u6h4n/StreamController/main.py", line 251, in <module>
    main()
    └ <function main at 0x7a1a773e6f20>

  File "/home/v4u6h4n/StreamController/main.py", line 239, in main
    create_global_objects()
    └ <function create_global_objects at 0x7a1a773e6520>

  File "/home/v4u6h4n/StreamController/main.py", line 121, in create_global_objects
    gl.window_grabber = WindowGrabber()
    │  │                └ <class 'src.backend.WindowGrabber.WindowGrabber.WindowGrabber'>
    │  └ None
    └ <module 'globals' from '/home/v4u6h4n/StreamController/globals.py'>

  File "/home/v4u6h4n/StreamController/src/backend/WindowGrabber/WindowGrabber.py", line 37, in __init__
    self.init_integration()
    │    └ <function WindowGrabber.init_integration at 0x7a1a773e4900>
    └ <src.backend.WindowGrabber.WindowGrabber.WindowGrabber object at 0x7a1a64dbf410>

  File "/home/v4u6h4n/StreamController/src/backend/WindowGrabber/WindowGrabber.py", line 47, in init_integration
    self.server = self.get_active_server()
    │             │    └ <function WindowGrabber.get_active_server at 0x7a1a773e4860>
    │             └ <src.backend.WindowGrabber.WindowGrabber.WindowGrabber object at 0x7a1a64dbf410>
    └ <src.backend.WindowGrabber.WindowGrabber.WindowGrabber object at 0x7a1a64dbf410>

  File "/home/v4u6h4n/StreamController/src/backend/WindowGrabber/WindowGrabber.py", line 43, in get_active_server
    return os.getenv("XDG_SESSION_TYPE").lower()
           │  └ <function getenv at 0x7a1ac3e099e0>
           └ <module 'os' (frozen)>

AttributeError: 'NoneType' object has no attribute 'lower'
2024-04-28 09:12:10.806 | TRACE    | __main__:<module>:254 - Reached end of main.py
```

**Comment by Core447** [comment]
Should be fixed with eab32ff5de2ad86879f3b6e34c99aa35f3eb25b6. Can you please try again?

**Comment by v4u6h4n** [comment]
Hey :-) alright yeah both are fixed now, thanks
